src/snowflake/cli/_plugins/nativeapp/manager.py

```diff
@@ -576,30 +580,36 @@ def create_app_package(self) -> None:
                 )
             )
 
-    def _expand_script_templates(
-        self, env: jinja2.Environment, jinja_context: dict[str, Any], scripts: List[str]
+    def _render_script_templates(
+        self,
+        render_from_str: Callable[[str, Dict[str, Any]], str],
+        jinja_context: dict[str, Any],
+        scripts: List[str],
     ) -> List[str]:
         """
         Input:
-        - env: Jinja2 environment
+        - render_from_str: function which renders a jinja template from a string and jinja context
         - jinja_context: a dictionary with the jinja context
-        - scripts: list of scripts that need to be expanded with Jinja
+        - scripts: list of script paths relative to the project root
         Returns:
-        - List of expanded scripts content.
+        - List of rendered scripts content
         Size of the return list is the same as the size of the input scripts list.
         """
         scripts_contents = []
         for relpath in scripts:
```

Comment:
nit: it's not always relative path, so maybe we can call it script_path

Reply:
I'm confused about this one: previously the function worked on `jinja.Environment`, which in both usages had `FileLoader` set on `project_root` (also in L599 `relpath` suggests that input paths should be relative). In current implementation the function won't work for global paths - should we change it? Do we currently have a use case for global paths?